naming/src/main/java/com/alibaba/nacos/naming/remote/rpc/handler/BatchInstanceRequestHandler.java

```diff
@@ -53,7 +53,7 @@ public BatchInstanceRequestHandler(EphemeralClientOperationServiceImpl clientOpe
     public BatchInstanceResponse handle(BatchInstanceRequest request, RequestMeta meta) throws NacosException {
         Service service = Service.newService(request.getNamespace(), request.getGroupName(), request.getServiceName(),
                 true);
-        InstanceUtil.batchSetInstanceIdIfEmpty(request.getInstances(), service.getGroupedServiceName());
+        InstanceUtil.batchSetInstanceIdIfEmpty(request.getInstances());
```

Comment:
the group name and service name in `Instance` maybe null and not equal with request, need use request first.